src/js/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @overview Fozzie header JavaScript behaviour.
+ *
+ * @module f-header
+ */
+
+import ready from 'lite-ready';
+
+
+const convertInputToButton = selector => {
+    const input = document.querySelector(selector);
+
+    if (input) {
+        const replaceTag = input.outerHTML.replace(/^<input/, '<button');
```

Comment:
Should we set the button `type` here otherwise it will default to a `submit` value? e.g `button type="button"` this way you can run the client side scripts safely without any default behavior going on in the background. :)

Reply:
Good shout :thumbsup:

Reply:
Done.